src/changelog.md

```diff
@@ -3,6 +3,9 @@
 ## 2019
 
 * **June 2019**
+  * Elasticsearch 7.2: We now support [Elasticsearch 7.2](/configuration/services/elasticsearch.md).
```

Comment:
If 5.2 and 5.4 are now deprecated, there should probably be an entry for them in the changelog too, like "[Elasticsearch](/configuration/services/elasticsearch.md) 5.2 and 5.4 support is now deprecated.

Reply:
Plus I think June and July got switched, and this should be these should be the most recent additions in **July 2019**.

Reply:
Fixed!